<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1580</width>
    <height>1030</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1580</width>
    <height>1030</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox_OI">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>427</width>
     <height>505</height>
    </rect>
   </property>
   <property name="title">
    <string>Oringinal Image</string>
   </property>
   <widget class="QPushButton" name="pushButton_OI_CF">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>438</y>
      <width>400</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>25</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Choose your Image.</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_OI">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>410</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>↗ </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_OI_i">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>28</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget_OI_WH" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>465</y>
      <width>150</width>
      <height>38</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_OI_WH_SUBTITLE">
       <property name="text">
        <string>W x H: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_OI_WH_NUM">
       <property name="text">
        <string>0 x 0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>pushButton_OI_CF</zorder>
   <zorder>label_OI_i</zorder>
   <zorder>toolButton_OI</zorder>
   <zorder>widget_OI_WH</zorder>
  </widget>
  <widget class="QPushButton" name="pushButton_convert">
   <property name="geometry">
    <rect>
     <x>449</x>
     <y>209</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Convert it into a grayscale image.</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>300</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_GA">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>426</width>
     <height>505</height>
    </rect>
   </property>
   <property name="title">
    <string>GRAY A.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label_GA">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>446</y>
      <width>400</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Gray = ( R + G + B ) / 3.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_GA">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>410</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>↗ </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_GA_i">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>28</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget_GA_WH" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>465</y>
      <width>150</width>
      <height>38</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_GA_WH_SUBTITLE">
       <property name="text">
        <string>W x H: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_GA_WH_NUM">
       <property name="text">
        <string>0 x 0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_GA_h" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>410</width>
      <height>280</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>391</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_GA_h">
      <property name="spacing">
       <number>6</number>
      </property>
     </layout>
    </widget>
   </widget>
   <zorder>label_GA</zorder>
   <zorder>label_GA_i</zorder>
   <zorder>toolButton_GA</zorder>
   <zorder>widget_GA_WH</zorder>
   <zorder>widget_GA_h</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBox_ADJ">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>681</width>
     <height>470</height>
    </rect>
   </property>
   <property name="title">
    <string>Adjustment</string>
   </property>
   <widget class="QGroupBox" name="groupBox_ADJ_TH">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Set the threshold value</string>
    </property>
    <widget class="QSlider" name="horizontalSlider_ADJ_TH">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>56</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_ADJ_TH">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>3</number>
     </property>
     <property name="intValue" stdset="0">
      <number>1</number>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_ADJ">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose the function you want: </string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_ADJ">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>34</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Brightness &amp; Constrast Adjustment</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Convert into binary image</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Histogram equalization</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Enlarge or Shrink &amp; Grayscale</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_ADJ_H">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>180</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Histogram equalization</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RESET">
    <property name="geometry">
     <rect>
      <x>568</x>
      <y>394</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_ADJ_SA" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>79</y>
      <width>221</width>
      <height>391</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_ADJ_SA_BC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>201</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Adjust the brightness and constrast.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QFrame" name="frame_ADJ_SA_BC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>201</width>
       <height>340</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_ADJ_SA_BC_B">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>280</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Brightness</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_ADJ_SA_BC_C">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>280</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Constrast</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSlider" name="verticalSlider_ADJ_SA_BC_C">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>80</width>
        <height>270</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-50</number>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QSlider" name="verticalSlider_ADJ_SA_BC_B">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>80</width>
        <height>270</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-255</number>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_ADJ_SA_BC_B">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>300</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_ADJ_SA_BC_C">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>300</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <zorder>verticalSlider_ADJ_SA_BC_B</zorder>
     <zorder>label_ADJ_SA_BC_B</zorder>
     <zorder>label_ADJ_SA_BC_C</zorder>
     <zorder>verticalSlider_ADJ_SA_BC_C</zorder>
     <zorder>lcdNumber_ADJ_SA_BC_B</zorder>
     <zorder>lcdNumber_ADJ_SA_BC_C</zorder>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_ADJ_ES" native="true">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>79</y>
      <width>331</width>
      <height>391</height>
     </rect>
    </property>
    <widget class="QFrame" name="frame_ADJ_ES">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>160</width>
       <height>341</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_ADJ_ES_SUBTITLE">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>280</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Enlarge &amp; Shrink</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSlider" name="verticalSlider_ADJ_ES">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>10</y>
        <width>151</width>
        <height>270</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_ADJ_ES">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>300</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_ADJ_ES_TITLE">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>161</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Adjust the spatial resolution.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_ADJ_ES_GS_TITLE">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>190</y>
       <width>141</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Grayscale</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_ADJ_ES_GS">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>220</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>8 bits</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4 bits</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2 bits</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 bits</string>
      </property>
     </item>
    </widget>
   </widget>
   <zorder>widget_ADJ_ES</zorder>
   <zorder>widget_ADJ_SA</zorder>
   <zorder>groupBox_ADJ_TH</zorder>
   <zorder>label_ADJ</zorder>
   <zorder>comboBox_ADJ</zorder>
   <zorder>pushButton_ADJ_H</zorder>
   <zorder>pushButton_RESET</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBox_GBA">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>550</y>
     <width>426</width>
     <height>470</height>
    </rect>
   </property>
   <property name="title">
    <string>GRAY B. after adjust.</string>
   </property>
   <widget class="QToolButton" name="toolButton_GBA">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>408</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>↗ </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_GBA_i">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>28</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget_GBA_WH" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>430</y>
      <width>150</width>
      <height>38</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_GBA_WH_SUBTITLE">
       <property name="text">
        <string>W x H: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_GBA_WH_NUM">
       <property name="text">
        <string>0 x 0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_GBA_h" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>410</width>
      <height>280</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>391</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_GBA_h"/>
    </widget>
   </widget>
   <zorder>label_GBA_i</zorder>
   <zorder>toolButton_GBA</zorder>
   <zorder>widget_GBA_WH</zorder>
   <zorder>widget_GBA_h</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBox_GAA">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>550</y>
     <width>426</width>
     <height>470</height>
    </rect>
   </property>
   <property name="title">
    <string>GRAY A. after adjust.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QToolButton" name="toolButton_GAA">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>407</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>↗ </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_GAA_i">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>28</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget_GAA_WH" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>430</y>
      <width>150</width>
      <height>38</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_GAA_WH_SUBTITLE">
       <property name="text">
        <string>W x H: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_GAA_WH_NUM">
       <property name="text">
        <string>0 x 0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_GAA_h" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>410</width>
      <height>280</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>391</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_GAA_h"/>
    </widget>
   </widget>
   <zorder>label_GAA_i</zorder>
   <zorder>toolButton_GAA</zorder>
   <zorder>widget_GAA_WH</zorder>
   <zorder>widget_GAA_h</zorder>
  </widget>
  <widget class="QCheckBox" name="checkBox_COMPARE">
   <property name="geometry">
    <rect>
     <x>449</x>
     <y>330</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Compare</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_GB">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>10</y>
     <width>426</width>
     <height>505</height>
    </rect>
   </property>
   <property name="title">
    <string>GRAY B.</string>
   </property>
   <widget class="QLabel" name="label_GB">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>446</y>
      <width>400</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>GRAY = 0.299 * R + 0.587 * G + 0.114 * B</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_GB">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>410</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>↗ </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_GB_i">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>28</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget_GB_WH" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>465</y>
      <width>150</width>
      <height>38</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_GB_WH_SUBTITLE">
       <property name="text">
        <string>W x H: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_GB_WH_NUM">
       <property name="text">
        <string>0 x 0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_GB_h" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>410</width>
      <height>280</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>391</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_GB_h"/>
    </widget>
   </widget>
   <zorder>label_GB_i</zorder>
   <zorder>label_GB</zorder>
   <zorder>toolButton_GB</zorder>
   <zorder>widget_GB_WH</zorder>
   <zorder>widget_GB_h</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
